### PR TITLE
Updated partners.yml file with program-areas field

### LINKED ISSUE #2459: Update partners.yml so each partner have a program-area property
### Dependency
This issue will be able to be closed when the following issues are completed
- ~#2271~
- ~#2384~

### Overview
We need to update our data files so liquid references become easier to implement. 

### Action Items
- [x] For each partner in the partners.yml file, add a new property called `program-area` and add the proper program-area value(s) in that field

<details>
<summary>Example for a partner with multiple program-areas</summary>

```
- name: Code for America
  image: /assets/images/about/sponsors/code-for-america.svg
  link: https://www.codeforamerica.org/
  program-area:
    - Citizen Engagement
    - Civic Tech Infrastructure
```

</details>

<details>
<summary>Example for a partner with a single program-area</summary>

```
- name: Ballotpedia
  image: /assets/images/about/partners/bp.png
  link: https://ballotpedia.org/Main_Page
  program-area:
    - Vote / Representation
```

</details>

### Resources/Instructions
The current partner.yml file does not have the program areas field.
The New version needs a program area's field.
This [comment](https://github.com/hackforla/website/issues/2459#issuecomment-974919932) has the replacement content.

[partners.yml](https://github.com/hackforla/website/blob/gh-pages/_data/internal/partners.yml)
[Program Area Audit](https://docs.google.com/spreadsheets/d/1wkzmWck-aGTzGxiryYEUhGbKRJX8rDDT3RLsAn-E-9o/edit#gid=0)

### PR DESCRIPTION
**Pull request by patrickohh**
Fixes #2459

### What changes did you make and why did you make them ?

  - Updated partners.yml file with appropriate program-areas field so liquid references are easier to implement

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
No apparent visual changes.

### CODE REVIEW

**Comment by MattPereira** [comment]
Review ETA: End of day 11/15/22
Availability: 7 - 10 PM

**Comment by jyaymie** [comment]
Review ETA: 11/15/22 5pm
Availability: 11/15/22 Flexible

File: _data/internal/partners.yml [hunk 1]
```diff
@@ -1,72 +1,121 @@
 - name: Los Angeles Department of Neighborhood Empowerment
   image: /assets/images/partners/empower-la.png
   link: https://empowerla.org/
+  program-areas:
```
Comment (jyaymie) [inline _data/internal/partners.yml]: Based on the issue, I think the property should be `program-area` (singular).

File: _data/internal/partners.yml [hunk 2]
```diff
@@ -1,72 +1,121 @@
...
   link: https://digitaldefensefund.org/
+  program-areas:
+    - Vote / Representation
 - name: City of Los Angeles Personnel Department
   image: /assets/images/partners/la-department-of-personnel.png
   link: https://personnel.lacity.org/
+  program-areas:
+    - Workforce Development
 - name: The Los Angeles County Homeless Initiative
   image: /assets/images/partners/la-county-homeless-initiative.png
```
Comment (jyaymie) [inline _data/internal/partners.yml:113]: I'm trying to figure out if the provided info in the issue's [comment](https://github.com/hackforla/website/issues/2459#issuecomment-974919932), last edited 22 days ago, is up to date. For example, The LA County Homeless Initiative's image, described in the comment as `real-help-lasting-change.svg` is different from what is found here, `la-county-homeless-initiative.png`. That image in the comment is not actually found in the current codebase, whereas the image here does exist. There are also differences in some of the names of the partners (e.g., `Metro` versus `Los Angeles Metro`).

**Review by jyaymie — CHANGES_REQUESTED** [review]
This looks great, @patrickohh! You didn't miss a single partner. One more (small) thing - I might remove the dropdowns in the PR if they're not being used.

**Comment by plang-psm** [comment]
Review ETA: 11/20/22
Availability: 6 hrs

File: _data/internal/partners.yml (hunk 1 above)
Comment (patrickohh) [reply _data/internal/partners.yml]: I'll be sure to update the field to this thanks!

File: _data/internal/partners.yml (hunk 2 above)
Comment (patrickohh) [reply _data/internal/partners.yml:113]: I will ask in the upcoming meeting about this. Thanks for the heads up.

File: _data/internal/partners.yml (hunk 1 above)
Comment (patrickohh) [reply _data/internal/partners.yml]: Alright just updated 'program-areas' to be 'program-area'

**Review by plang-psm — APPROVED** [review]
Looks good! Noticed you fixed the 'program-areas' to 'program-area', and the site ran normally after the changes. Good work!!

**Comment by patrickohh** [comment]
Some differences in partners.yml compared to the information in this [comment](https://github.com/hackforla/website/issues/2459#issuecomment-974919932):
1. `name: Los Angeles Department of Neighborhood Empowerment` (current) ; `name: Empower LA` (comment)
2. `name: Los Angeles Metro` (current); `name: Metro` (comment)
3. `name: Los Angeles Department of Transportation` (current); `name: LADOT` (comment)   
4. `image: /assets/images/partners/la-county-homeless-initiative.png` (current); `image: /assets/images/partners/real-help-lasting-change.svg` (comment)

**Comment by Suman2795** [comment]
ETA:16/11/2022
Availability: 3 hours

**Review by Suman2795 — COMMENTED** [review]
There are differences in the image description of comment and the file but the program areas field added is done correct.

**Review by MattPereira — APPROVED** [review]
I agree with @Suman2795 . The `program-area` property has been correctly implemented as instructed by the issue. Concerns about differences between the [comment](https://github.com/hackforla/website/issues/2459#issuecomment-974919932) and the current codebase(for some of the partners' `name` and `image` properties) may be warranted, but seem to fall outside the scope of #2459. Given that the `link` property still matches between both sources, I think its safe to assume patrick has properly correlated the `program-area` field value(s) for each partner. 

Great job combing through the comment and codebase to document all of the discrepencies @patrickohh ! :+1:

File: _data/internal/partners.yml (hunk 2 above)
Comment (blulady) [reply _data/internal/partners.yml:113]: We discussed this at a meeting, we're going to confirm the name changes with Bonnie on Sunday before merging.

**Comment by blulady** [comment]
~All changes were intentional continue as instructed.~ revised by bonnie on 2022-11-26

File: _data/internal/partners.yml (hunk 2 above)
Comment (ExperimentsInHonesty) [reply _data/internal/partners.yml:113]: Actually @patrickohh and @blulady I agree the changes are outside the scope.  I think we should accept the changes as is.  And fix anything else that we want changed after the pr has been accepted.  i will add a note to the issue to indicate the steps that come after.

**Comment by ExperimentsInHonesty** [comment]
@hackforla/website-merge  team, please accept this pr if you are satisfied that the program-area changes have happened.  I'll leave a note on the issue about the remaining items that need to be audited before they can be addressed (in a new issue).

**Comment by jdingeman** [comment]
Merging PR without "waiting for requirements to be met" since I can't seem to find the unresolved conversation. Any changes requested look like they were resolved in the thread.

**Pull request merged**